core/src/main/java/com/scalar/db/api/Put.java

```diff
@@ -42,10 +43,6 @@ public class Put extends Mutation {
 
   private final Map<String, Column<?>> columns;
 
-  private boolean implicitPreReadEnabled;
-
-  private boolean insertModeEnabled;
```

Comment:
Removed the flags for implicit pre-read and insert mode. Instead, use the operation attributes for them.